File: 프로그래머스/0/120906. 자릿수 더하기/자릿수 더하기.cs
using System;

public class Solution {
    public int solution(int n) 
    {
        string numStr = n.ToString();
        int numSum = 0;
        for(int i=0; i<numStr.Length; i++) {
            numSum += Int32.Parse(numStr[i].ToString());
        }
        return numSum;
    }
}

File: 프로그래머스/0/120906. 자릿수 더하기/자릿수 더하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int n) 
{
    string nstr = to_string(n);
    int numSum = 0;
    for(char &c : nstr)
    {
        numSum += c - '0';
    }
    return numSum;
}